lib/src/generateZodClientFromOpenAPI.ts

```diff
@@ -3,19 +3,21 @@ import path from "node:path";
 import type { OpenAPIObject } from "openapi3-ts";
 import { capitalize, pick } from "pastable/server";
 import type { Options } from "prettier";
-import { match } from "ts-pattern";
+import { match, P } from "ts-pattern";
 
 import { getHandlebars } from "./getHandlebars";
 import { maybePretty } from "./maybePretty";
 import type { TemplateContext } from "./template-context";
 import { getZodClientTemplateContext } from "./template-context";
 
+export const PredefinedTemplates = ["schemas-only", "export-schemas"] as const;
+
 type GenerateZodClientFromOpenApiArgs<TOptions extends TemplateContext["options"] = TemplateContext["options"]> = {
     openApiDoc: OpenAPIObject;
-    templatePath?: string;
+    templatePath?: typeof PredefinedTemplates | string;
```

Comment:
I think it should be 

```ts
templatePath?: typeof PredefinedTemplates[number] | (string & {});
```
<img width="696" alt="Screenshot 2023-05-04 at 10 19 30" src="https://user-images.githubusercontent.com/47224540/236148855-8636fead-f983-4b18-a4b8-f8b8a1d6c70b.png">